src/ResourceManager/Insights/Commands.Insights/help/Add-AzureRmLogProfile.md

```diff
@@ -8,7 +8,9 @@ schema: 2.0.0
 # Add-AzureRmLogProfile
 
 ## SYNOPSIS
-Creates a log profile.
+Creates a new activity log profile, there can be only one log profile per subscription. This profile is used to either archive the activity log to a storage account or stream it to an Azure event hub. Only a single standard storage account (premium storage account is not supported) per subscription is supported. It could either be of type ARM or Classic. If it's logged to a storage account, the cost of storing the activity log is billed at normal standard storage rates. 
```

Comment:
> there can be only one log profile per subscription

We are increasing the limit. Can we not include the limit there? We don't want to come change a comment in the sdk every time we increase it.

> Only a single standard storage account (premium storage account is not supported) per subscription is supported

This also seems to be due to the limit. Can we remove this?